script/download_folio_record.rb

```diff
@@ -2,11 +2,17 @@
 # frozen_string_literal: true
 
 require 'json'
-require_relative '../config/boot'
```

Comment:
this loads a bunch of things, including honeybadger, which prints its startup message and pollutes the output so it can't be parsed as JSON. if there's any easier way to disable that, i'd be happy to do it.